### PR TITLE
modificate AllMovies - add movies from API

### PR DESCRIPTION
**Pull request by dembskipawel**

### CODE REVIEW

File: src/views/AllMovies.vue [hunk 1]
```diff
@@ -1,21 +1,61 @@
...
+      return `${hours}:${minutes}`;
+    },
+  },
+  async created() {
+    try {
+      const response = await axios.get(`http://localhost:3000/movies`);
+      this.movies = response.data;
+    } catch (e) {
+      this.errors.push(e);
+    }
```
Comment (rossinek) [inline src/views/AllMovies.vue:53]: really nice to see a proper error handling! ❤️

File: src/views/AllMovies.vue [hunk 2]
```diff
@@ -1,21 +1,61 @@
...
 </script>
 
-<style >
-  @import '@/assets/styles/reset.css';
-  @import '@/assets/styles/styles.css';
-</style>
\ No newline at end of file
+<style>
+@import "@/assets/styles/reset.css";
+@import "@/assets/styles/styles.css";
```
Comment (rossinek) [inline src/views/AllMovies.vue:60]: You don' need to repeat global styles in every component. You can import it just once e.g. in `App.vue` or `main.js`.

File: src/assets/styles/styles.css [hunk 3]
```diff
@@ -178,6 +190,18 @@ body {
 
     }
 
+    .main_intro_links_bar {
```
Comment (rossinek) [inline src/assets/styles/styles.css:193]: personally I would suggest to move component related styles directly to the `.vue` file with this component. It is much easier to manage styles if all component related styles are inside this component file 😉

**Review by rossinek — APPROVED** [review]

File: src/views/AllMovies.vue (hunk 2 above)
Comment (dembskipawel) [reply src/views/AllMovies.vue:60]: ok i will fix this code

File: src/assets/styles/styles.css (hunk 3 above)
Comment (dembskipawel) [reply src/assets/styles/styles.css:193]: ok thanks for the suggestions

**Pull request merged**